src/e2e-test/features/mssql/RunTime.feature

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Mssql
+Feature: Mssql - Verify Mssql source data transfer to BigQuery
+
+  @MSSQL_SOURCE @MSSQL_DELETE @BQ_SINK_TEST
+  Scenario: To verify replication of snapshot data from Mssql to BigQuery successfully
+    Given Open DataFusion Project with replication to configure pipeline
+    When Enter input plugin property: "name" with pipelineName
```

Comment:
Are we updating this name property with a random name for each pipeline ? 
If not we should do it.

Reply:
yes we are updating with a random name for each pipeline.